crates/webidl-tests/lib.rs

```diff
@@ -1 +1,3 @@
 // intentionally left blank
+
+// QUESTION FOR REVIEWER: WHY? (I'll include this context in the comment)
```

Comment:
Ah this is just because Cargo needs to have *something* like a library or a binary, so this is just a library to pacify cargo, but the library isn't actually used by any of the tests currently

Reply:
Oh thanks! Even though there's a `main.rs` file?